README.md

```diff
@@ -36,6 +36,8 @@ If you are new to Graviton and want to understand how to identify target workloa
 |---	|---	|---	|
 |Instances	|[M6g/M6gd](https://aws.amazon.com/ec2/instance-types/m6/), [C6g/C6gd/C6gn](https://aws.amazon.com/ec2/instance-types/c6/), [R6g/R6gd](https://aws.amazon.com/ec2/instance-types/r6/), [T4g](https://aws.amazon.com/ec2/instance-types/t4), [X2gd](https://aws.amazon.com/ec2/instance-types/x2/), [G5g](https://aws.amazon.com/ec2/instance-types/g5g/), and [Im4gn/Is4gen](https://aws.amazon.com/ec2/instance-types/i4g/)	|[C7g](https://aws.amazon.com/ec2/instance-types/c7g/)	|
 |Core	|[Neoverse-N1](https://developer.arm.com/documentation/100616/0301)	|[Neoverse-V1](https://developer.arm.com/documentation/101427/latest/)	|
+|Frequency	|2500MHz	|2600MHz	|
+|Turbo supported	|No	|No	|
```

Comment:
the new page needs to be linked to the main page with some kind of blurb. 

The page itself shouldn't be about x86 vs graviton differences, but just generally about graviton monitoring where you can mention there are differences.

Reply:
Done.